src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -1830,7 +1830,7 @@ internal static bool MayUseCallForStructMethod(MethodSymbol method)
         {
             Debug.Assert(method.ContainingType.IsVerifierValue(), "this is not a value type");
 
-            if (!method.IsMetadataVirtual())
+            if (!method.IsMetadataVirtual() || method.IsStatic)
```

Comment:
To make sure I understand the change: it looks like this lets us use 'call' instruction when calling a method that implements a static abstract method. Is that correct?

Reply:
>To make sure I understand the change: it looks like this lets us use 'call' instruction when calling a method that implements a static abstract method. Is that correct?

At the moment, this chnage is not observable in any way, but static methods can be invoked only by 'call' instruction.